docs/_templates/auto_api_templates/python/module.rst

```diff
@@ -3,26 +3,88 @@
+   |Experiment| #This is an object to which you log any piece of information you consider to be important during your run. Interaction with the experiment feels similar to interacting with a singleton dictionary object. Neptune gives you all the freedom - you simply log metrics, images, text, and everything else to particular names and those objects are sent to the application. You can have one or multiple experiments in one script. You can re-instantiate the experiments you have created in the past and update them.
    |Notebook| #Contains all the information about a Neptune Jupyter Notebook.
    |Git Info| #Keeps information about the Git repository in an experiment.
 
+Learn how to get started with logging and managing experiment data using Neptune :ref:`here <guides-logging-and-managing-experiment-results>`.
+
 {% endif %}
 
-:mod:`{{ obj.name }}`
-======={{ "=" * obj.name|length }}
+{% if 'neptunecontrib' == obj.name %}
+
+This library contains community extensions. This is what you use to integrate Neptune with other frameworks. Check all the integrations Neptune supports :ref:`here <integrations-index>`.
+
+{% endif %}
+
+{% if 'neptune_tensorboard' == obj.name %}
+
+This library supports packages and functions for Neptune's integration with TensorBoard. Read how to integrate Neptune with TensorBoard :ref:`here <integrations-tensorboard>`.
+
+{% endif %}
+
+{% if 'neptune.experiments' == obj.name %}
+
+An Experiment is everything that you log to Neptune, beginning at ``neptune.create_experiment()`` and ending when script finishes or when you explicitly stop the experiment with ``neptune.stop`` (reference docs: :meth:`~neptune.experiments.Experiment.stop`).
+
+Creating experiments is easy:
+
+.. code-block:: python3
+
+    # Set project
+    neptune.init('my_workspace/my_project')
+
+    # Create new experiment
+    neptune.create_experiment()
+
+You can now log various data to the experiment including metrics, losses, model weights, images, predictions and much more. Have a look at the complete list of :ref:`what you can log <what-you-can-log>` to the experiment.
+
+Besides logging data, you can also :ref:`download experiment data <guides-download_data>` to you local machine or :ref:`update an existing experiment <update-existing-experiment>` even when it's closed.
+
+.. note::
+
+    ``neptune.log_metric('some_name', some_value)`` is for tracking all numeric values to Neptune (metric, loss, score, variances, etc.). Learn, what else can be tracked to experiment from :ref:`this list <what-you-can-log>`.
+
+{% endif %}
+
+{% if 'neptune.projects' == obj.name %}
+
+A Project is a **collection of Experiments**, created by user (or users) assigned to the project.
+
+You can log experiments to the project or fetch all experiments that satisfy some criteria.
+
```

Comment:
I'd divide it in paragraphs and explain that. 

neptune.init() sets a global project. 

but you can use project object explicitly

project = neptune.init()

You can create experiments in the project explicitly 

project.create_experiment()

or use the global context and simply go

neptune.create_expriment()

you can access experiments from the project by doing

project.get_experiments() 

or that you can show the leaderboard dataframe of the project by going

project.get_leaderboard()